### PR TITLE
Add sample for trigger using http_destination

### PR DESCRIPTION
**Pull request by nlopezgi**
Add sample for trigger using http_destination

- [x] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [x] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [x] [Generated Terraform providers](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/generate-providers.md), and ran [make test and make lint](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/run-provider-tests/#run-unit-tests) in the generated providers to ensure it passes unit and linter tests.
 - [x] [Ran](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/run-provider-tests.md) relevant acceptance tests using my own Google Cloud project and credentials (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [x]  Read [Write release notes](https://googlecloudplatform.github.io/magic-modules/contribute/release-notes/) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
eventarc: added support for `http_endpoint.uri` and `network_config.network_attachment` to `google_eventarc_trigger`
clouddeploy: added support for `canary_revision_tags`, `prior_revision_tags`, `stable_revision_tags`, and `stable_cutback_duration` to `google_clouddeploy_delivery_pipeline`
```

### CODE REVIEW

File: mmv1/third_party/terraform/services/eventarc/resource_eventarc_trigger_test.go.erb [hunk 1]
```diff
@@ -53,6 +53,44 @@ func TestAccEventarcTrigger_channel(t *testing.T) {
...
+
+	// Need to have the full network attachment name in the format project/{project_id}/regions/{region_id}/networkAttachments/{networkAttachmentName}
+	fullFormNetworkAttachmentName := fmt.Sprintf("projects/%s/regions/%s/networkAttachments/%s", envvar.GetTestProjectFromEnv(), envvar.GetTestRegionFromEnv(), networkAttachmentName)
+
+	context := map[string]interface{}{
+		"region":          region,
+		"project_name":    envvar.GetTestProjectFromEnv(),
+		"service_account": envvar.GetTestServiceAccountFromEnv(t),
+                "network_attachment": fullFormNetworkAttachmentName,
+
```
Comment (NickElliot) [inline mmv1/third_party/terraform/services/eventarc/resource_eventarc_trigger_test.go.erb:73]:
```suggestion
                "network_attachment": fullFormNetworkAttachmentName,
                "random_suffix": acctest.RandString(t, 10),
```

**Review by NickElliot — CHANGES_REQUESTED** [review]
need this to be updated!

**Comment by nlopezgi** [comment]
> need this to be updated!
fixed

**Review by NickElliot — APPROVED** [review]
Unrelated test failure, LGTM! Thanks for handling the long contribution process!

**Pull request merged**